test/jsonwp-proxy/proxy-specs.js

```diff
@@ -134,18 +134,19 @@ describe('proxy', () => {
       e.value.should.eql({message: 'Invisible element'});
       e.status.should.equal(11);
     });
-    it('should throw when a command fails with a 200', async () => {
+    it('should return response body when a command fails with a 200', async () => {
       let j = mockProxy({sessionId: '123'});
+      let res = null;
       let e = null;
       try {
-        await j.command('/element/200/text', 'GET');
+        res = await j.command('/element/200/text', 'GET');
       } catch (err) {
         e = err;
       }
-      should.exist(e);
-      e.message.should.contain('Original error: Invisible element');
-      e.value.should.eql({message: 'Invisible element'});
-      e.status.should.equal(11);
+      should.exist(res);
+      res.value.should.eql({message: 'Invisible element'});
+      res.status.should.eql(11);
+      should.not.exist(e);
```

Comment:
This check for `e`, along with the try/catch block are not necessary, since this does not throw anymore. We just need

```
let j = mockProxy({sessionId: '123'});
let res = await j.command('/element/200/text', 'GET');
res.value.should.eql({message: 'Invisible element'});
res.status.should.eql(11);
```